message_definitions/v1.0/development.xml

```diff
@@ -14,6 +14,15 @@
         <description>True if the data from this sensor is being actively used by the flight controller for guidance, navigation or control.</description>
       </entry>
     </enum>
+    <enum name="RADIO_RC_CHANNELS_FLAGS" bitmask="true">
+      <description>RADIO_RC_CHANNELS flags (bitmask).</description>
+      <entry value="1" name="RADIO_RC_CHANNELS_FLAGS_FAILSAFE">
+        <description>Failsafe is active.</description>
+      </entry>
+      <entry value="2" name="RADIO_RC_CHANNELS_FLAGS_FRAME_MISSED">
+        <description>Indicates that the current frame has not been received. Channel values are frozen.</description>
+      </entry>
+    </enum>
```

Comment:
Do we need both of these?  A "channels are valid" flag might be sufficient?  Could we clarify or rename entry 2 to mean exactly that, with any other bits set jsut being additional information, perhaps?

The ArduPilot implementation was only looking at one of the two existing bits, but it seems to me we should be ignoring input if either is set.  I'd prefer a singlet bit to indicate whether we should be trusting the channel values ata all.

Reply:
we sure don't necessarily need FRAME_MISSED, but I considered it usefull (even if it may not be used in AP code)
it's kind of mirroing sbus
we could also rename it to CHANNELS_INVALID, even though I'm not sure I see what it improves.

Reply:
per dev call decission, flag will be removed